src/mount/mount.ceph.c

```diff
@@ -289,7 +289,7 @@ static int parse_arguments(int argc, char *const *const argv,
 			}
 			*opts = argv[i];
 		}
-		else {
+		else if (strcmp("-s", argv[i])) {
```

Comment:
if we intend to support "-s", maybe we should ignore all unknown arguments if "-s" is passed in (the sloppy mode), instead of just ignoring "-s", which IMHO, is a workaround.